test/create-adapter.test.ts

```diff
@@ -186,6 +187,17 @@ describe("adapter creation =>", () => {
 				await expectSuccess("adapter_TS_React", answers);
 			});
 
+			it("Adapter, Tab, TypeScript React", async () => {
+				const answers: Answers = {
+					...baseAnswers,
+					adminFeatures: ["tab"],
+					es6class: "yes",
+					adminReact: "yes",
+					tabReact: "yes",
+				};
+				await expectSuccess("adapter_tab_TS_React", answers);
```

Comment:
Since this test is aimed at checking the differences `tabReact` introduces, I'd add a file filter here.

Reply:
Not sure what you mean by this. How do I add a file filter?

Reply:
Here's an example:
https://github.com/ioBroker/create-adapter/blob/c3441f28de83f8c9ca15b53497dfdc771db3777e/test/create-adapter.test.ts#L373-L386
This only outputs the files that are interesting for this test. Usually makes sense for smaller features.

Reply:
Updated in 1125166, and added one more test for mixed Admin in HTML and Tab in React, which seems to be a common use case.